src/sss_client/nss_passwd.c

```diff
@@ -468,5 +469,6 @@ enum nss_status _nss_sss_endpwent(void)
     }
```

Comment:
I think this check (and similar checks in other files) doesn't make sense anymore since `errno` will be overwritten unconditionally and thus can be removed.